torch/distributed/fsdp/_fully_shard/_fully_shard.py

```diff
@@ -159,7 +208,7 @@ def fully_shard(
     modules = (
         (module,) if isinstance(module, nn.Module) else tuple(_get_root_modules(module))
     )
-    state = fully_shard.state(modules[0])
+    state = fully_shard.state(modules[0])  # type: ignore[attr-defined] # [1]
```

Comment:
What is the `[1]` here for?

Reply:
It corresponds to the reason mentioned at L83. The format oddity comes from the fact that mypy doesn't like additional comment before the `# type: ignore` or anything directly after it (hence the extra #)